pkg/skaffold/schema/latest/config.go

```diff
@@ -195,8 +195,9 @@ type HelmRelease struct {
 	Version           string                 `yaml:"version,omitempty"`
 	SetValues         map[string]string      `yaml:"setValues,omitempty"`
 	SetValueTemplates map[string]string      `yaml:"setValueTemplates,omitempty"`
-	Wait              bool                   `yaml:"wait,omitempty"`
 	RecreatePods      bool                   `yaml:"recreatePods,omitempty"`
+	Wait              bool                   `yaml:"wait,omitempty"`
```

Comment:
It may make more sense to have a `Flags []string` field, that way a change doesn't need to be made to skaffold every time a new flag needs to be added.

```yaml
flags:
- --wait
- --timeout=50
```
and these can be directly appended to args in `helm.go`